retriever/closing.py

```diff
@@ -1,11 +1,11 @@
 import pandas as pd
 import requests
 
-from last import last
-from logger import Logger
+from utils.last import last
+from utils.logger import Logger
```

Comment:
Don't know why the module `utils` has to be included here, while it is not necessary for `Logger` or `last`

Reply:
Is only a way to clarify the imports, where they are come from. In this case from a subpackage.